diff --git a/todolist/settings.py b/todolist/settings.py
index c4fc492..fe3e6bb 100644
--- a/todolist/settings.py
+++ b/todolist/settings.py
@@ -23,7 +23,7 @@ SECRET_KEY = "@e2(yx)v&tgh3_s=0yja-i!dpebxsz^dg47x)-k&kq_3zf*9e*"
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['*']
 
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
 
